xchange-stream-hitbtc/src/main/java/info/bitrich/xchangestream/hitbtc/dto/HitbtcWebSocketOrderBook.java

```diff
@@ -35,16 +44,22 @@ private void createFromLevels(HitbtcWebSocketOrderBookTransaction orderbookTrans
     }
 
     sequence = orderbookTransaction.getParams().getSequence();
+    timestamp = parseISOTimestampInUTC(orderbookTransaction.getParams().getTimestamp());
```

Comment:
This parsing is built into Java. You could just use `ZonedDateTime.parse(orderbookTransaction.getParams().getTimestamp()).toInstant().toEpochMilli()`

Reply:
I didn't know that. Thanks. Everyday we learn something new :-)
I'll change, test and submit correction.

Reply:
Doesn't seem to work with UTC timestamp. Getting the following exception:
`Text '2020-05-03T18:26:28.714Z' could not be parsed: Unable to obtain ZonedDateTime from TemporalAccessor: {MicroOfSecond=714000, MilliOfSecond=714, NanoOfSecond=714000000, InstantSeconds=1588530388},ISO of type java.time.format.Parsed`
I've tried other built-in formatters which seems to support milliseconds, but having same issue. A'm I missing something?

Reply:
Ignore the previous comment, I had to supply date format with time zone. Added it - works perfect. I'll commit soon.

Reply:
I got interested in why you needed a formatter. Might possibly be [this JDK bug](https://bugs.openjdk.java.net/browse/JDK-8041360) because it works for me  on JDK11 without any special handing:
![2020-05-03 (1)](https://user-images.githubusercontent.com/6483013/80923274-909dab00-8d7a-11ea-9db5-2bd8ed2cce96.png)

Reply:
No, I don;t think is related. That bug seems was fixed 8u20. Actually, checking history of changes, I guess I messed up with my code, which caused that. I had something like in this shell:
![image](https://user-images.githubusercontent.com/2891073/80925275-4b7f7600-8d86-11ea-97b0-044c100a0bf8.png)

So, I changed the code to short and simple version as you suggested. Thanks.